portfolio/src/main/webapp/index.html

```diff
@@ -4,6 +4,27 @@
     <meta charset="UTF-8" />
     <title>Giovanni's Portfolio</title>
     <link rel="stylesheet" href="style.css" />
+
+    <style>
+      /* Always set the map height explicitly to define the size of the div
+       * element that contains the map. */
+      #map {
+        height: 100%;
+      }
+      /* Optional: Makes the sample page fill the window. */
+      html,
+      body {
+        height: 100%;
+        margin: 0;
+        padding: 0;
+      }
+    </style>
```

Comment:
[Style](https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns): Separate structure from presentation from behavior.

Please move the styling to your CSS file

Reply:
You may want to alphabetize CSS properties and use shorthand ones (comments from PR #8) while you're at it